### PR TITLE
:octocat: PR: Submission CREATE animals TABLE (Vet-clinic)

### PR DESCRIPTION
**Pull request by mahabubx7**
### :octocat: PR: Submission CREATE animals TABLE (Vet-clinic)

- [x] Created `vet-clinic` database.
- [x] Created `animals` table.
- [x] Inserted provided data into table.
- [x] Made required quires.

#### Screenshot of query & their results below.
Screenshot 2023-06-05 193243
Screenshot 2023-06-05 193343

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @mahabubx7 ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] Kindly use the correct [GitFlow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). You are using GitHub Flow at the moment and this project requires the use of [GitFlow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) instead. Make sure your feature branch is merging into the dev branch.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahabubx7** [comment]
> ## Hi @mahabubx7 ✋
> Good job so far!
> 
> Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ### This project will be marked as Invalid and won't affect the number of your weekly reviews.
> * [x]  Kindly use the correct [GitFlow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). You are using GitHub Flow at the moment and this project requires the use of [GitFlow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) instead. Make sure your feature branch is merging into the dev branch.
> 
> Cheers, and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks for your kind correction. I was just forgot about that 😸

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int not null,
```
Comment (BregornOriginal) [inline schema.sql]: - [OPTIONAL] Great work creating an animal table. It would be great to add a `PRIMARY KEY(id)`

**Review by BregornOriginal — APPROVED** [review]
Hi @mahabubx7  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (7)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR title and description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]

**Pull request merged**